packages/flutter/lib/src/push/parse_push.dart

```diff
@@ -0,0 +1,74 @@
+part of flutter_parse_sdk_flutter;
+
+class ParsePush {
+  static final ParsePush instance = ParsePush._internal();
+  static String keyType = "gcm";
+  static String keyPushType = 'pushType';
+
+  factory ParsePush() {
+    return instance;
+  }
+
+  ParsePush._internal();
+
+  Future<void> initialize(firebaseMessaging) async {
+    // get GCM token
+    firebaseMessaging.getToken().asStream().listen((event) async {
+      // set token in ParseInstallation
```

Comment:
```suggestion
      // Set token in installation
```

Reply:
The concept of installation cannot be understood where it is being set.
Are you sure about this?